src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineVirtualSpoutHandler.java

```diff
@@ -43,6 +48,23 @@
     // Logger for logging
     private static final Logger logger = LoggerFactory.getLogger(SidelineVirtualSpoutHandler.class);
 
+    private PersistenceAdapter persistenceAdapter;
```

Comment:
Do you think we should rename this to SidelinePersistenceAdapter to avoid confusion?

Reply:
I go back and force on this... I mean we definitely can if you think it'd help.